mapflow/functional/service/data_catalog.py

```diff
@@ -160,3 +167,21 @@ def selected_mosaic(self) -> Optional[MosaicReturnSchema]:
         if not first:
             return None
         return first[0]
+    
+    def selected_images(self, limit=None) -> List[MosaicReturnSchema]:
```

Comment:
As far as I see, there are the same changes in another PR, which will be hard to merge later. There are ways to rebase to use the same commit, but currently I would recommend to work on such changes one-by-one. Or we could discuss it later.

Reply:
I think for this branch I will leave it like that (changed this in #111) and after all other flaws will be fixed, I will create a new branch from current version of `feature/data-catalog-api` and manually transfer needed chenges there from `image_info`. 
Futher on I will try to avoid these situations.